addons/base_automation/models/base_automation.py

```diff
@@ -594,6 +594,15 @@ def _get_eval_context(self, payload=None):
             eval_context['payload'] = payload
         return eval_context
 
+    def _get_evaluated_domain(self, domain):
+        self.ensure_one()
+        domain = domain.replace('.to_utc()', '')
```

Comment:
This cannot be merged.
You are trying to manipulate code that is evaluated dynamically. Looks like you want to ignore some function that is possibly available in JS, but not in python. Following this patch, we could introduce only security issues and end with a language which is a mix of python and javascript.